include/networkit/numerics/LAMG/LevelHierarchy.hpp

```diff
@@ -40,15 +42,16 @@ class LevelHierarchy {
                              const std::vector<EliminationStage<Matrix>> &coarseningStages);
     void addAggregationLevel(const Matrix &A, const Matrix &P, const Matrix &R);
     void setLastAsCoarsest();
-    inline DenseMatrix &getCoarseMatrix() { return coarseLUMatrix; }
+    inline const DenseMatrix &getCoarseMatrix() const { return coarseLUMatrix; }
 
     inline count size() const {
         return levelType.size() + 1; // elimination + aggregation levels + finestLevel
     }
 
     LevelType getType(index levelIdx) const;
     Level<Matrix> &at(index levelIdx);
-    double cycleIndex(index levelIdx);
+    const Level<Matrix> &at(index levelIdx) const;
```

Comment:
I added a const version because the new LAMG (#1212) uses `const LevelHierarchy` objects in the `solve` functions, where access to the `at` version is required.